projects/storefrontapp-e2e-cypress/cypress/integration/b2b/regression/saved-cart/b2b-saved-cart.e2e-spec.ts

```diff
@@ -0,0 +1,161 @@
+import * as savedCart from '../../../../helpers/b2b/b2b-saved-cart';
+import { viewportContext } from '../../../../helpers/viewport-context';
+import * as sampleData from '../../../../sample-data/b2b-saved-cart';
+
+context('B2B - Saved Cart', () => {
+  viewportContext(['mobile', 'desktop'], () => {
+    before(() => {
+      cy.window().then((win) => win.sessionStorage.clear());
+      cy.window().then((win) => win.localStorage.clear());
+      cy.clearLocalStorageMemory();
```

Comment:
Not sure if we need to clear localstorage as it's getting cleared with every test suite.

Reply:
I like having them just for just in case, but I can remove it